README.md

```diff
@@ -159,6 +159,40 @@ Stacktrace:
 [...]
 ```
 
+### Breakpoint at specific location
+It is sometimes useful to programmatically decide when to break and drop to the debugger to inspect your surroundings; this is for instance done in Matlab/Octave with the `keyboard` command. You can use the `@bp` macro in JuliaInterpreter to do this:
+```
```

Comment:
Can we make this a doctest?

Reply:
Probably not -- it requires user input in the debug mode.

Reply:
Ah, good point.